Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml

```
@@ -6,7 +6,7 @@
   description: A checkerboard. Pieces included!
   components:
   - type: Sprite
-    sprite: Objects/Fun/Tabletop/chessboard.rsi
+    sprite: SS220/Objects/Fun/Tabletop/chessboard.rsi
```

```suggestion
    sprite: SS220/Objects/Fun/Tabletop/chessboard.rsi #SS220-Tabletop-Resprite
```
Ну и такой же коммент везде ниже, где ты в офф файлах менял путь. Это так, на всякий случай